torch/_prims/__init__.py

```diff
@@ -929,7 +931,7 @@ def _fill_aten(a: Tensor, value: NumberType) -> Tensor:
 # div prim performs truncation division on integer inputs
 #   and true division for floating and complex inputs
 def _div_aten(a, b):
-    is_integral = isinstance(a, (bool, int)) or (
+    is_integral = isinstance(a, (bool, int, torch.SymIntNode)) or (
```

Comment:
bleh, I guess it feels pointless to make a dedicated named value for `(bool, int, torch.SymIntNode)`

Reply:
actually it's kind of weird the lint is complaining about this site, hmmmm